src/apps/ItemSheet.js

```diff
@@ -43,4 +45,12 @@ export default class ItemSheet extends SvelteApplication {
       }
     });
   }
+
+  static getSheetComponent(type) {
+    switch (type) {
+      case 'background': return BackgroundSheetComponent;
+      case 'culture': return CultureSheetComponent;
+      default: return ItemSheetComponent;
+    }
+  }
```

Comment:
Typically, I'd advise just avoiding `switch` statements. In almost all cases, they end up being longer and requiring more boilerplate code than just using basic conditionals.
```js
static getSheetComponent(type) {
  if (type === 'background') return BackgroundSheetComponent;
  if (type === 'culture') return CultureSheetComponent;
  return ItemSheetComponent;
}
```
The `switch` version is more lines and more characters, and it doesn't offer any additional clarity.

Reply:
When this expands to additional items, the `switch` still won't make any sense. Using an object to map types to classes will be the better approach at that point.